src/coreclr/vm/interoplibinterface.cpp

```diff
@@ -17,6 +19,17 @@
 using CreateObjectFlags = InteropLib::Com::CreateObjectFlags;
 using CreateComInterfaceFlags = InteropLib::Com::CreateComInterfaceFlags;
 
+void* GetCurrentCtxCookieWrapper()
```

Comment:
Thanks for creating a wrapper. In this case, the function should be put within the anonymous namespace below. Also, please add the `STATIC_CONTRACT_WRAPPER;` to the beginning of the function.

Reply:
Just call `return` not the `RETURN` macro which is for more complex Contract scenarios.

Reply:
Ok